test/likes.js

```diff
@@ -1,7 +1,149 @@
-// import chai from 'chai';
-// import chaiHttp from 'chai-http';
-// import index from '../index';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
 
-// chai.use(chaiHttp);
+chai.use(chaiHttp);
+chai.should();
 
-// const [expect] = [chai.expect];
+
+chai.use(chaiHttp);
+
+describe('/likes', () => {
+  const fakeToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjg3LCJ1c2VybmFtZSI6ImpvZWRlb28ifQ.E1SOwBDBq5oAJRAgziQFKm6PeJUcnYgVXusw3EC54jc';
+  let userToken;
+  before(async () => {
+    const userData = {
+      email: 'publisherb@heimdal.com',
+      password: '12345678heimdal',
+    };
+    const userResponse = await chai.request(app)
+      .post('/api/v1/auth/login')
+      .send(userData);
+    const { token } = userResponse.body;
+    userToken = token;
+  });
+
+  it('should return status code 400 when user is not logged in', async () => {
```

Comment:
Nice work Victor. Kindly cross check your test description for it to match with the status code. 
When a user is not logged in it should be 401. Thanks